### DISCUSSION TITLE
show number of matches on website beatmap listing

### DISCUSSION THREAD

**Discussion by yukifunea** [ppy/osu#30528] [Ideas] [answered]
i want this 
![image](https://github.com/user-attachments/assets/f0b356b3-3d1e-42dd-91a8-f6049e304008)

on the website here as well 
![image](https://github.com/user-attachments/assets/cd21893c-e976-4429-be3d-805c82a919d7)

**Comment by cl8n** [comment, marked as answer]
tracked at https://github.com/ppy/osu-web/issues/6175 although that hasn't really gone anywhere for a while.
